lucupy/minimodel/observation.py

```diff
@@ -244,13 +244,56 @@ def instrument(self) -> Optional[Resource]:
         """
         def check_instrument(r: Optional[Resource]):
             if r is not None:
-                # To avoid a circular import.
-                from lucupy.observatory.abstract import ObservatoryProperties
-                return ObservatoryProperties.is_instrument(r)
+                # # TODO: Why comparing to ObservatoryProperties instead of checking resource.type??
```

Comment:
ObservatoryProperties makes the model observatory agnostic. I don't think is the highest of priorities right now, so let's move forward with this.